src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --font-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.page-header {
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo-img {
+  margin: 7px 0;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.page-navigation {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.li-item {
+  padding: 0 10px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  text-transform: uppercase;
+  text-align: center;
+  display: inline-block;
+  margin: auto;
+}
+
+.is-active {
+  color: var(--font-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: var(--font-color);
+}
+
+.nav-link:hover {
+  color: var(--font-color);
+}
+
+.li-last {
+  padding-right: 0;
+}
+
+.li-first {
```

Comment:
FYI: you can use `:first-child` or `:last-child` structural pseudoclasses for such purposes